client/assets/stylesheets/style-bare.scss

```diff
@@ -0,0 +1,8 @@
+// External Dependencies
+@import 'vendor';
+
+// Color Schemes
+@import '~@automattic/calypso-color-schemes/src/calypso-color-schemes';
```

Comment:
We need these now _only_ for dev badge's colors... should we import it conditionally only in development-mode? That might lead to surprise breakage in production.

Performance penalty including them in the bundle even if we don't use them isn't huge so prolly ok to leave.

More over, there's a good chance that we'll end up using them eventually even if we'd mostly rely on Gutenberg color variables otherwise.